qemu_v8.mk

```diff
@@ -356,6 +396,21 @@ optee-os: optee-os-common
 
 optee-os-clean: optee-os-clean-common
 
+################################################################################
+# Hafnium
+################################################################################
+
+HAFNIUM_EXPORTS = PATH=$(HAFNIUM_PATH)/prebuilts/linux-x64/clang/bin:$(HAFNIUM_PATH)/prebuilts/linux-x64/dtc:$(PATH)
+
+.hafnium_checkout:
+	(cd $(HAFNIUM_PATH) && git submodule init && git submodule update)
+	touch .hafnium_checkout
+
+hafnium_base: .hafnium_checkout | $(OUT_PATH)
+	$(HAFNIUM_EXPORTS) $(MAKE) -C $(HAFNIUM_PATH) $(HAFNIUM_FLAGS) all
+
+hafnium: hafnium_base
```

Comment:
Why is `hafnium_base` needed? Why not depend on the generated binary? I.e.,:
```make
hafnium: $(HAFNIUM_BIN)

$(HAFNIUM_BIN): .hafnium_checkout | $(OUT_PATH)
	$(HAFNIUM_EXPORTS) $(MAKE) -C $(HAFNIUM_PATH) $(HAFNIUM_FLAGS) all
```

Reply:
Good idea, I'll update.